src/samples/http/client.c

```diff
@@ -169,7 +169,7 @@ create_header_params(struct sol_http_params *params, const char *value)
     param.value.key_value.key = key;
     param.value.key_value.value = val;
 
-    if (!sol_http_param_add_copy(params, param)) {
+    if (sol_http_params_add_copy(params, param) < 0) {
```

Comment:
On all those bool->int conversions where we got also logging, a ``sol_util_strerrora()`` usage won't hurt :)